.github/actions/composite/setupNode/action.yml

```diff
@@ -16,40 +16,24 @@ runs:
       uses: actions/cache@v3
       with:
         path: node_modules
-        key: ${{ runner.os }}-node-modules-${{ hashFiles('package-lock.json') }}
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('package-lock.json', 'patches/**') }}
 
     - id: cache-desktop-node-modules
       uses: actions/cache@v3
       with:
         path: desktop/node_modules
-        key: ${{ runner.os }}-desktop-node-modules-${{ hashFiles('desktop/package-lock.json') }}
-
-    - name: Check if patch files changed
-      id: patchCheck
-      shell: bash
-      run: |
-        set -e
-        git fetch origin main --no-tags --depth=1
-        if git diff --name-only origin/main HEAD | grep -q '\.patch'; then
-          echo 'CHANGES_IN_PATCH_FILES=true' >> "$GITHUB_OUTPUT"
-        else
-          echo 'CHANGES_IN_PATCH_FILES=false' >> "$GITHUB_OUTPUT"
-        fi
```

Comment:
Curious whether we are currently using `CHANGES_IN_PATCH_FILES` elsewhere? Given that we persisted it to a var it suggests that we were?